astacus/node/api.py

```diff
@@ -133,6 +133,16 @@ def cassandra(req: Union[ipc.NodeRequest, ipc.CassandraStartRequest], subop: ipc
         raise HTTPException(status_code=501, detail="Cassandra support is not installed")
     if not n.state.is_locked:
         raise HTTPException(status_code=409, detail="Not locked")
+    if not n.config.cassandra:
+        raise HTTPException(status_code=500, detail="Cassandra node configuration not found")
+    config_path = n.config.cassandra.client.config_path
+    if not config_path:
+        raise HTTPException(status_code=500, detail="Expected Cassandra client configuration path value not found")
+    if not config_path.exists():
+        raise HTTPException(
+            status_code=500,
+            detail=f"Cassandra client configuration file not found at {config_path!r}",
+        )
```

Comment:
This seems slightly unnecessary; the Path could be converted to `pydantic.FilePath` which ensures it exists. 

This is also somewhat unexpected place to check for it, though, as in theory some of the operations work without file too? (e.g. they require just ability to get client, which also works using host+port)?

Reply:
Converted the field to the `FilePath` type, removed the extra checks and added a test covering the failure scenario